moped-database/views/project_list_view.sql

```diff
@@ -82,6 +81,45 @@ moped_proj_components_subtypes AS (
     LEFT JOIN moped_components AS mc ON mpc.component_id = mc.component_id
     WHERE mpc.is_deleted = false
     GROUP BY mpc.project_id
+),
+
+project_district_association AS (
+
+    WITH
+    project_council_district_map AS (
+        SELECT DISTINCT
+            project_id,
+            unnest(council_districts) AS council_district
+        FROM project_geography
+    ),
```

Comment:
one other issue related to `uniform_features`: it appears that there is a bug in that view (courtesy of yours truly 🤦 ) such that deleted council districts are not excluded from the view. i was able to reproduce this by creating a project with two components, each in a different district, and then deleting one of the components.

Reply:
Thanks again for this feedback. Commit [ead951d](https://github.com/cityofaustin/atd-moped/pull/1274/commits/ead951d74cfc7ec582a0bbac108d5d92e0c696d4) addresses it, I believe.

Reply:
thanks for that!